Tests/com/drew/imaging/jpeg/JpegMetadataReaderTest.java

```diff
@@ -57,6 +59,24 @@ public void testExtractXmpMetadata() throws Exception
         assertNotNull(directory);
     }
 
+    @Test
+    public void testFStopFormatting() throws Exception
+    {
+        Locale previousLocale = Locale.getDefault();
+
+        // Setting this locale would cause an f-number with a comma:
+        Locale.setDefault(Locale.forLanguageTag("nl-NL"));
+
+        try {
+            Metadata metadata = JpegMetadataReader.readMetadata(new File("Tests/Data/withExif.jpg"));
+            final ExifSubIFDDirectory subIFDDirectory = metadata.getFirstDirectoryOfType(ExifSubIFDDirectory.class);
+            assertEquals("f/4.0", subIFDDirectory.getDescription(ExifDirectoryBase.TAG_FNUMBER));
```

Comment:
Thanks for the clarification. Because the library exposes so many numeric values I've had requests and patches in the past that actually force the output to consider the current locale. In general I think that's the safest approach. If you require a different formatting, you can temporarily override the locale to `en-GB` or something like that when computing descriptions. Alternatively get the value as an int/double/etc and go from there.

Until now this isn't something that's been formalised. I should specify that the library supports locales for all output strings, and then ensure that all formatting actually does that.